### ISSUE TITLE
Rule Change: function-call-argument-newline

### ISSUE THREAD

**Issue by Pzixel** [eslint/eslint#15632]
### What rule do you want to change?

function-call-argument-newline

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
function test (
    a,
    b, c, d,
    e, f
) {
}
```

### What does the rule currently do for this code?

Nothing

### What will the rule do after it's changed?

It should format it according to `function-call-argument-newline`'s options

### Participation

- [X] I am willing to submit a pull request to implement this change.

### Additional comments

Maybe there is some plugin or existing rule for it but I found none. 
It might be more profitable to enable a new rule like `function-declaration-argument-newline` but since new rules policy is what it is I propose it as an option to existing rule.

I believe it's so basic it deserves to be included in core rules.

**Comment by mdjermanovic** [comment]
Hi @Pzixel, thanks for the issue!

 Per our [Stylistic Rule Updates](https://github.com/eslint/eslint#stylistic-rule-updates) policy, we are not adding any more options to stylistic rules unless it is the only way to fix a bug or support a new ES feature. I think the best course of action would be to create a custom rule for function parameters and publish it in a plugin.
